tests/conftest.py

```diff
@@ -92,49 +96,79 @@ def workflow_id(indico_client, dataset_obj):
             )
     return workflow_id
 
+
 @pytest.fixture(scope="module")
 def module_submission_ids(workflow_id, indico_client, pdf_filepath):
     workflow_wrapper = Workflow(indico_client)
     sub_ids = workflow_wrapper.submit_documents_to_workflow(workflow_id, [pdf_filepath])
     workflow_wrapper.wait_for_submissions_to_process(sub_ids)
     return sub_ids
 
+
 @pytest.fixture(scope="function")
 def function_submission_ids(workflow_id, indico_client, pdf_filepath):
     workflow_wrapper = Workflow(indico_client)
     sub_ids = workflow_wrapper.submit_documents_to_workflow(workflow_id, [pdf_filepath])
     workflow_wrapper.wait_for_submissions_to_process(sub_ids)
     return sub_ids
 
+
 @pytest.fixture(scope="module")
 def wflow_submission_result(indico_client, module_submission_ids) -> dict:
     workflow_wrapper = Workflow(indico_client)
     return workflow_wrapper.get_submission_results_from_ids(
         [module_submission_ids[0]], timeout=90
     )[0]
-  
+
+
 @pytest.fixture(scope="session")
 def model_name():
     return MODEL_NAME
 
+
 @pytest.fixture(scope="session")
 def ondoc_ocr_object(indico_client, pdf_filepath):
-    job = indico_client.call(DocumentExtraction(files=[pdf_filepath], json_config={"preset_config": "ondocument"}))
+    job = indico_client.call(
+        DocumentExtraction(
+            files=[pdf_filepath], json_config={"preset_config": "ondocument"}
+        )
+    )
     job = indico_client.call(JobStatus(id=job[0].id, wait=True))
     extracted_data = indico_client.call(RetrieveStorageObject(job.result))
     return extracted_data
 
+
 @pytest.fixture(scope="session")
 def standard_ocr_object(indico_client, pdf_filepath):
-    job = indico_client.call(DocumentExtraction(files=[pdf_filepath], json_config={"preset_config": "standard"}))
+    job = indico_client.call(
+        DocumentExtraction(
+            files=[pdf_filepath], json_config={"preset_config": "standard"}
+        )
+    )
     job = indico_client.call(JobStatus(id=job[0].id, wait=True))
     extracted_data = indico_client.call(RetrieveStorageObject(job.result))
     return extracted_data
 
+
 @pytest.fixture(scope="session")
 def doc_extraction_standard(indico_client):
     return DocExtraction(indico_client)
 
+
 @pytest.fixture(scope="session")
 def snapshot_csv_path(testdir_file_path):
     return os.path.join(testdir_file_path, "data/snapshots/snapshot.csv")
+
+
+@pytest.fixture(scope="session")
+def snapshot_classes():
```

Comment:
I think this fixture should be within the tests/snapshots module, really unlikely it is going to be used outside of that. my feeling is that base level conftest.py should only be for fixtures that are going to be used across sub-modules

Reply:
or possibility that it will be in the future

Reply:
otherwise that file is going to get enormous

Reply:
other than that- everything looks good to me

Reply:
Made a conftest for snapshots